sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1863,6 +1870,7 @@ export class SearchClient<T> implements IndexDocumentsClient<T> {
 export interface SearchClientOptions extends ExtendedCommonClientOptions {
     // @deprecated
     apiVersion?: string;
+    audience?: SearchAudience;
```

Comment:
I'm concerned about taking the enum type here, since presumably there will be additional national clouds or perhaps even unlisted ones that can be passed as well. I feel like this should be `string` but have a ref comment that mentions using the `SearchAudience` enum for convenience.

@bterlson @joheredi thoughts?

Reply:
@xirzec You are right. Modified to make it string and changed the other name to include Known